roofit/roofitcore/src/RooDirItem.cxx

```diff
@@ -77,7 +77,7 @@ void RooDirItem::appendToDir(TObject* obj, Bool_t forceMemoryResident)
     if (notInMemNow) {
       gDirectory->cd(pwd) ;    
     }
-
+*/
```

Comment:
leaving commented code in is typically not considered good practice (also not sure whether it was intentional)

Reply:
Yes, I should take it out.